TwitchLib.Api/TwitchAPI.cs

```diff
@@ -35,6 +34,40 @@ public TwitchAPI(ILoggerFactory loggerFactory = null, IRateLimiter rateLimiter =
             V5 = new V5.V5(loggerFactory, _rateLimiter, Settings, _http);
             ThirdParty = new ThirdParty.ThirdParty(Settings, _rateLimiter, _http);
             Undocumented = new Undocumented(Settings, _rateLimiter, _http);
+
+            settings.PropertyChanged += Settings_PropertyChanged;
+        }
+
+        private void Settings_PropertyChanged(object sender, System.ComponentModel.PropertyChangedEventArgs e)
+        {
+            switch (e.PropertyName)
+            {
+                case "AccessToken":
+                    V5.Settings.AccessToken = Settings.AccessToken;
+                    Helix.Settings.AccessToken = Settings.AccessToken;
+                    break;
```

Comment:
Just like the other cases, we're breaking here. Meaning it'll get out of the Switch case, resulting in the ``` throw new System.NotImplementedException();``` being called. The throw should be in the default case, or all breaks must be replaced with ```return;```